packages/react/src/components/controls/MediaDeviceSelect.tsx

```diff
@@ -49,6 +49,7 @@ export interface MediaDeviceSelectProps extends React.HTMLAttributes<HTMLUListEl
   onActiveDeviceChange?: (deviceId: string) => void;
   onDeviceListChange?: (devices: MediaDeviceInfo[]) => void;
   initialSelection?: string;
+  exactMatch?: boolean;
```

Comment:
Maybe a short docstring to explain what exactMatch does and what happens if there is no exact match.